include/s3select.h

```diff
@@ -1218,6 +1226,27 @@ void push_between_filter::builder(s3select* self, const char* a, const char* b)
   self->getAction()->exprQ.push_back(func);
 }
 
+void push_not_between_filter::builder(s3select* self, const char* a, const char* b) const
+{
+  const std::string not_between_function("#not_between#");
```

Comment:
please use `static constexpr const std::string_view not_between_function("#not_between#");`
need to replace `c_str()` with `data()`